tgui/packages/tgui/interfaces/ListInput.js

```diff
@@ -62,6 +62,14 @@ export const ListInput = (props, context) => {
                 e.preventDefault();
                 acquireHotKey(KEY_DOWN);
                 acquireHotKey(KEY_UP);
+
+                for (let i = 65; i <= 90; i++) {
+                  acquireHotKey(i);
+                }
+                for (let i = 48; i <= 57; i++) {
+                  acquireHotKey(i);
+                }
+
```

Comment:
where is release

Reply:
it will be released when the user selects or cancels